cf-deployment.yml

```diff
@@ -11,7 +11,6 @@ addons:
 - name: loggregator_agent
   include:
     stemcell:
-    - os: ubuntu-xenial
     - os: ubuntu-bionic
```

Comment:
Why not rip out bionic as well?

Reply:
Well if anyone still needs to use Bionic with own ops files, it would be somewhat difficult to re-add "ubuntu-bionic" for all the bosh addons. We could remove this in one of the next releases.